src/main/scala/com/framex/core/ElemX.scala

```diff
@@ -61,6 +60,14 @@ object ElemX {
     }
   }
 
+  def castToDouble(data: String) : ElemX = {
+    try {
+      ElemX(ExDouble(data.toDouble))
+    } catch {
+      case _ => ElemX(ExString(data))
```

Comment:
Why `castToDouble` return `ElemX(ExString(.))` when error happend?

Reply:
It's now used in reading csv.

When you read data from csv, the default ElemX for each element is ExString. It's used to "check if one element can be cast to numeric or not, if success, then cast to double, or just use default string type".

Reply:
Maybe we need a good name for this method, or refactor it's logic...

Reply:
What if we let the user decide the element type...

Reply:
Like `readCsv[Int,Double,String]("/path")` or something else...

Reply:
Let' me think a few well...